cli/main.go

```diff
@@ -26,6 +35,55 @@ func main() {
 		EnableBashCompletion:   true,
 		UseShortOptionHandling: true,
 		Commands: []*cli.Command{
+			{
+				Name:      "find-stale-pods",
+				Args:      true,
+				Usage:     "Locate stale pods, whose balances have deviated by more than 5%% due to beacon slashing.",
+				UsageText: "./cli find-stale-pods <args>",
+				Flags: []cli.Flag{
+					EXEC_NODE_FLAG,
+					BEACON_NODE_FLAG,
+				},
```

Comment:
Verbose flag is not included here

Reply:
verbose is global (included in the top-level args at the bottom of this fn)